tests/playwright/utils/deploy_utils.py

```diff
@@ -100,3 +100,12 @@ def deploy(location: str, app_name: str, app_file_path: str) -> str:
     else:
         raise ValueError("Unknown deploy location. Cannot deploy.")
     return url
+
+
+# Since connect parses python packages, we need to get latest version of shiny on main
+def tweak_requirements_txt(app_file_path: str) -> None:
+    requirements_file_path = os.path.join(app_file_path, "requirements.txt")
+    git_cmd = subprocess.run(["git", "rev-parse", "main"], stdout=subprocess.PIPE)
```

Comment:
I think we don't want to base this off of `main` -- for example, i fthere's a PR, we want to use that branch. I think we can just use `HEAD` here.

```suggestion
    git_cmd = subprocess.run(["git", "rev-parse", "HEAD"], stdout=subprocess.PIPE)
```